cms/djangoapps/contentstore/course_group_config.py

```diff
@@ -8,8 +8,8 @@
 
 from django.utils.translation import ugettext as _
 from contentstore.utils import reverse_usage_url
-from xmodule.partitions.partitions import UserPartition
-from xmodule.partitions.partitions_service import get_all_partitions_for_course, MINIMUM_STATIC_PARTITION_ID
+from xmodule.partitions.partitions import UserPartition, MINIMUM_STATIC_PARTITION_ID
```

Comment:
Note to reviewers: I moved ENROLLMENT_TRACK_PARTITION_ID and MINIMUM_STATIC_PARTITION_ID into `partitions` to minimize dependencies on `partitions_service` (which currently depends on django).